cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,23 @@
+import time
+
+from django.core.management import BaseCommand
+from django.db import connections
+from psycopg2 import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for db to start", ending="\n")
+        db = None
+        attempts = 5
```

Comment:
The number of attempts is hardcoded to 5. Consider making this configurable via command-line options or environment variables to provide more flexibility.